### DISCUSSION TITLE
How to stream nginx-proxy logs to a file?

### DISCUSSION THREAD

**Discussion by nguyenleminhquan** [nginx-proxy/nginx-proxy#2373] [Q&A]
Hi everyone,

I want to use Goaccess to analyze nginx-proxy logs, but currently there seems to be no option to stream logs to file.
Have you ever used Goaccess to analyze nginx-proxy logs? If yes, how can you do that?

Thank you for reading my question.

**Comment by shiz0** [comment]
Hi,
I am not familiar with Goaccess, but Docker has various options to log to files in different formats and even remote locations.
Here's a guide to Docker logging: https://sematext.com/guides/docker-logs/
And the respective Docker Docs section: https://docs.docker.com/config/containers/logging/configure/

**Comment by prashker** [comment]
@nguyenleminhquan Did you get anywhere with a GoAccess solution that uses logs produced by nginx-proxy?
